### DISCUSSION TITLE
installation problem

### DISCUSSION THREAD

**Discussion by AYSEGULKZC** [NASA-LIS/LISF#1480] [Q&A]
I am having problems with the installation.

 I followed the installation in the link "https://nasa-lis.github.io/LISF/public_testcase_walkthrough/public_testcase_walkthrough.html".

 I used this command " git clone -b support/lisf-public-7.4 github:NASA-LIS/LISF.git" but it gave an error.

 " git clone -b support/lisf-public-7.4 https://github.com/NASA-LIS/LISF.git" command didn't work either.

When I entered "git clone -b support/lisf-public-7.4 git@github.com:NASA-LIS/LISF.git", I saw the text "ECDSA key fingerprint is...". 

I tried to download LISmaster.zip, I got the error that the file is corrupt and cannot be opened.

When I load "LISF-x86_64.AppImage", I see the following text: 
"./LISF-x86_64.AppImage -h
Usage: LISF-x86_64.AppImage [options] [-- LISF options]

   where [options] are:
   -h show this help message
   -v show revision info about LISF
   -i show configuration info about LISF
   -a <LDT|LIS|LVT> specify which app to run

   where [LISF options] are options to pass to LISF;
   these must be preceded by --

   Examples:
      LISF-x86_64.AppImage -a LDT ldt.config
      mpirun -n N LISF-x86_64.AppImage -a LIS -- --file lis.config
      LISF-x86_64.AppImage -a LVT lvt.config"

I'm confused about the installation. I need your immediate help. 

My main goal is to run a VIC model. I am also trying to create my own files. I wanted to complete a testcase first to understand it better. I would be very grateful if you have any advice for me.

**Comment by jvgeiger** [comment]
Hello,

Sorry for the delay.

This command

```
git clone -b support/lisf-public-7.4 github:NASA-LIS/LISF.git
```

assumes that you have created an account on GitHub and added your SSH key per the instructions in our "Working with GitHub" document, https://nasa-lis.github.io/LISF/working_with_github/working_with_github.html.  Sorry for the confusion.  I will update the walkthrough document as soon as possible.


This command

```
git clone -b support/lisf-public-7.4 https://github.com/NASA-LIS/LISF.git
```

works for me.  Would you please post any error messages that you get when you try to rerun that command?


> I tried to download LISmaster.zip, ...

For the Public Testcase Walkthrough, please use the Public 7.4.4 release, not the master branch.  The Public 7.4.4 release also has an AppImage container that you may use.

Running `./LISF-x86_64.AppImage -h` will produce the help message that you saw.  This suggests that the AppImage will run on your computer.  To use it to run, for example the "Run LDT - The Parameter Processing Step" from the Walkthrough, replace

```
% ./LDT ldt.config.noah36_params
```

with

```
% LISF-x86_64.AppImage -a LDT ldt.config.noah36_params
```

Thanks,
Jim
